resource_control: support config background tasks resource limit (#18945)
pingcap/docs: tidb-resource-control.md

#### `BACKGROUND` parameters

- `TASK_TYPES`: specifies the task types that need to be managed as background tasks. Use commas (`,`) to separate multiple task types.

> - `UTILIZATION_LIMIT`: specifies the maximum resource percentage(0-100) of each TiKV instance that background tasks can consume. By default, the resource granted for background tasks are calculated based on the total resource quota and current foreground workload dynamically, if `UTILIZATION_LIMIT` is configured, the resource granted for background tasks will not exceed this limit.

Reviewer: ```suggestion
- `UTILIZATION_LIMIT`: specifies the maximum resource percentage (0-100) of each TiKV instance that background tasks can consume. By default, the resource granted for background tasks are calculated based on the total resource quota and the current foreground workload dynamically. If `UTILIZATION_LIMIT` is configured, the resource granted for background tasks will not exceed this limit.
```

#### `BACKGROUND` parameters

- `TASK_TYPES`: specifies the task types that need to be managed as background tasks. Use commas (`,`) to separate multiple task types.

> - `UTILIZATION_LIMIT`: specifies the maximum resource percentage (0-100) of each TiKV instance that background tasks can consume. By default, the resource granted for background tasks are calculated based on the total resource quota and the current foreground workload dynamically. If `UTILIZATION_LIMIT` is configured, the resource granted for background tasks will not exceed this limit.

Reviewer 2: ```suggestion
- `UTILIZATION_LIMIT`: limits the maximum percentage  (0-100) of resources that background tasks can consume on each TiKV node. By default, TiKV calculates the available resources for background tasks based on the total resources of the node and the resources currently occupied by the foreground tasks. If `UTILIZATION_LIMIT` is configured, the resource allocated to background tasks will not exceed this limit.
```

> - `UTILIZATION_LIMIT`: limits the maximum percentage  (0-100) of resources that background tasks can consume on each TiKV node. By default, TiKV calculates the available resources for background tasks based on the total resources of the node and the resources currently occupied by the foreground tasks. If `UTILIZATION_LIMIT` is configured, the resource allocated to background tasks will not exceed this limit.

Author: ```suggestion
- `UTILIZATION_LIMIT`: limits the maximum percentage (0-100) of resources that background tasks can consume on each TiKV node. By default, TiKV calculates the available resources for background tasks based on the total resources of the node and the resources currently occupied by the foreground tasks. If `UTILIZATION_LIMIT` is configured, the resource allocated to background tasks will not exceed this limit.
```

#### Examples

> 1. Modify the `default` resource group and mark `br` and `ddl` as background tasks and set background tasks resource limit to 30%.

Reviewer: ```suggestion
1. Modify the `default` resource group and mark `br` and `ddl` as background tasks, and set background tasks resource limit to 30%.
```

#### Examples

> 1. Modify the `default` resource group and mark `br` and `ddl` as background tasks, and set background tasks resource limit to 30%.

Reviewer 2: ```suggestion
1. Modify the `default` resource group, mark `br` and `ddl` as background tasks, and set the background tasks resource limit to 30%.
```
